static/styles/right_sidebar.css

```diff
@@ -199,7 +199,6 @@ $user_status_emoji_width: 24px;
     right: 0;
     text-align: center;
     vertical-align: middle;
-    border-left: 1px solid hsl(0, 0%, 88%);
```

Comment:
We're now avoiding lines between items in the header bar. This removes some of that border.